dashboard/src/shared/utils.ts

```diff
@@ -1,5 +1,5 @@
 export function escapeRegExp(str: string) {
-  return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
+  return str.replace(/[-[\]/{}()*+?.\\^$|]/g, "\\$&");
```

Comment:
Hmm, this change isn't a mechanical one? What was the motivation here - looks like the regex for replacement was escaping the actual regex chars when it didn't need to? I can't produce any difference for different inputs (though this would be much easier to see if we already had a test and showed it didn't change).

Reply:
nm, I see it is just a lint warning for `no-useless-escape`.